README.md

```diff
@@ -45,14 +45,23 @@ the application is being run. This is best practice as described by
 | `LOG_LEVEL`                        | [log level](https://github.com/trentm/node-bunyan#levels)                | Depends on `NODE_ENV`   |            |
 | `JSON_FILE_URL`                    | publicly available URL of JSON data                                      |                         | yes        |
 | `ES_HOST`                          | host name of Elasticsearch server                                        |                         | yes        |
-| `ES_INDEX`                         | Elasticsearch configuration to read, currently only 'profiles' available | profiles                |            |
+| `ES_INDEX`                         | Elasticsearch configuration to read, currently only 'profiles' available |                         | yes        |
```

Comment:
The description for `ES_INDEX` isn't correct. It should be something along the lines of `Name of index to contain data`. For another PR though.